### ISSUE TITLE
Nuget package

### ISSUE THREAD

**Issue by maitredede** [m-e-conroy/angular-dialog-service#41]
Hi,
Your work seems to be nice :)
Is it possible to make a Nuget package to install your lib (and its dependencies) easily ?
Thanks.

**Comment by joaomello** [comment]
Hi,
@MaitreDede in last news from Microsft, either at Nuget GUI at Visual Studio advice to not user for front-end packages see: 
![Example] (http://blogs.msdn.com/cfs-filesystemfile.ashx/__key/communityserver-blogs-components-weblogfiles/00-00-00-63-56/2514.nugetIncompatible.png)

and at new scaffold from asp.net 5, uses bower and npm for this packages.

:smiley:

**Comment by MaitreDede** [comment]
Yes, but when I created this issue, it was pre-asp.net 5. And there are still projects that are not on this plateform. That why I asked :)
